source/elements/oneMKL/source/domains/rng/device_routines/device-bibliography.rst

```diff
@@ -0,0 +1,100 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_bibliography:
+
+Bibliography
+============
+
+
+.. container::
+
+
+   For more information about the RNG functionality, refer to the
+   following publications:
+
+
+   -
+
+
+      .. container::
+
+
+         **RNG**
+
+         [Bratley88]
```

Comment:
I am not sure if we need all these bibliography in device routines
Does it make sense to have a common bibliography for all RNG?

Reply:
United bibliography